packages/esm-tb-app/src/views/dashboard/summary-tiles/tb-summary-tiles.component.tsx

```diff
@@ -1,11 +1,30 @@
 import { OHRIProgrammeSummaryTiles } from '@ohri/openmrs-esm-ohri-commons-lib';
 import React, { useEffect, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { fetchMambaReportData } from '../../../../../../packages/esm-commons-lib/src/api/api';
 
 function TbSummaryTiles({ launchWorkSpace }) {
   const { t } = useTranslation();
-  const [activeDSClientsCount, setActiveDSClientsCount] = useState(35);
-  const [activeDRClientsCount, setActiveDRClientsCount] = useState(5);
+  const [activeDSClientsCount, setActiveDSClientsCount] = useState(null);
+  const [activeDRClientsCount, setActiveDRClientsCount] = useState(null);
+  
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const dsCount = await fetchMambaReportData('total_active_ds_cases');
+        const drCount = await fetchMambaReportData('total_active_dr_cases');
```

Comment:
This can be bundled in a Promise.all

```
const [dsCount, drCount] = await Promise.all([
  fetchMambaReportData('total_active_ds_cases'),
  fetchMambaReportData('total_active_dr_cases'),
]);
```

Reply:
Done @pirupius Please review again. Thank u